ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml

```diff
@@ -62,6 +62,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )cinder-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build04
+    context: ci/prow/cinder-operator-build-deploy-kuttl
+    decorate: true
+    labels:
+      ci.openshift.io/generator: prowgen
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openstack-k8s-operators-cinder-operator-master-cinder-operator-build-deploy-kuttl
+    optional: true
```

Comment:
don't we want to block on kuttl test failures though?

Reply:
I think we are keeping all the kuttl workflows non voting for now, maybe till they are stable or till the demo?
Example:
- https://github.com/openshift/release/pull/37275/files#diff-285068ae4683b9b46983eb9940594dd82e41f5d6e83ba2bf7331a1616d9c145fR97
- https://github.com/openshift/release/pull/37103/files#diff-f519f0f7af9c285191571fe9eca8d6a45324fa0a235bb8538b1bd593890ec147R97